openmdao/components/tests/test_ks_comp.py

```diff
@@ -170,36 +170,35 @@ def test_upper(self):
         prob = om.Problem()
         model = prob.model
 
-        model.add_subsystem('px', om.IndepVarComp('x', val=np.array([5.0, 4.0])))
         model.add_subsystem('comp', om.ExecComp('y = 3.0*x', x=np.zeros((2, )), y=np.zeros((2, ))))
+        model.set_input_defaults('comp.x', np.array([5.0, 4.0]))
```

Comment:
`set_val` here as well and throughout this doc

Reply:
fixed